api/envoy/api/v2/route/route.proto

```diff
@@ -1075,6 +1076,12 @@ message Tracing {
   // :ref:`HTTP Connection Manager <config_http_conn_man_runtime>`.
   // Default: 100%
   type.FractionalPercent overall_sampling = 3;
+
+  // A list of custom tags with unique tag name to create tags for the active span,
+  // applied after merging with the connection manager :ref:`tracing configuration
+  // <envoy_api_field_config.filter.network.http_connection_manager.v2.HttpConnectionManager.tracing.custom_tags>`,
+  // here takes priority if configured with the same tag name.
```

Comment:
nit: This sentence is hard to read. I recommend putting this part as a separate sentence.